### PR TITLE
fix(orchestrator): typos in auditLog messages

### PR DESCRIPTION
**Pull request by gciavarrini**

### CODE REVIEW

**Review by batzionb — APPROVED** [review]

File: plugins/orchestrator-backend/src/service/router.ts [hunk 1]
```diff
@@ -876,7 +876,7 @@ function setupInternalRoutes(
     'abortWorkflow',
     async (c, _req, res, next) => {
       const instanceId = c.request.params.instanceId as string;
-      const endpointName = 'getInstanceById';
+      const endpointName = 'abortWorkflow';
       const endpoint = `/v2/instances/${instanceId}/abort`;
```
Comment (rgolangh) [inline plugins/orchestrator-backend/src/service/router.ts:880]: /v2/instances here also looks wrong @batzionb

Comment (rgolangh) [reply plugins/orchestrator-backend/src/service/router.ts:880]: https://github.com/janus-idp/backstage-plugins/pull/2141

Comment (batzionb) [reply plugins/orchestrator-backend/src/service/router.ts:880]: @gciavarrini

Comment (gciavarrini) [reply plugins/orchestrator-backend/src/service/router.ts:880]: @rgolangh i wouldn't say "wrong" but inconsistent. Thanks for the fix :)

**Pull request merged**